packages/client/src/components/block-defaults/checkbox-block/CheckboxBlock.tsx

```diff
@@ -52,6 +46,9 @@ export const CheckboxBlock: BlockComponent = observer(({ id }) => {
                     const value = e.target.checked;
                     // update the value
                     setData('value', value);
+                    debounce(() => {
```

Comment:
This violates the rules of Hooks. See https://react.dev/warnings/invalid-hook-call-warning